### ISSUE TITLE
Uncompressed data size mismatch (nodejs)

### ISSUE THREAD

**Issue by gregoire-astruc** [Stuk/jszip#184]
Hello,

I was trying to extract a file and ran into the following exception:

```console
$ node index.js 
zipper/node_modules/jszip/lib/zipEntry.js:67
                throw new Error("Bug : uncompressed data size mismatch");
```

This seems close to #97.

The incriminating archive is [KW_Rocketry.zip](https://kerbalstuff.com/mod/67/KW%20Rocketry/download/2.6d2)

The code stripped down to the file which makes the error appear:

```js
"use strict";
var fs = require("fs");
var JSZip = require("jszip");

fs.readFile("KW_Rocketry-2.6d2.zip", function (err, data) {
    if (err) throw err;
    var zip = new JSZip(data);

    zip.file('KW Release Package v2.6d2 (Open this, don\'t extract it)/GameData/KWRocketry/Parts/Aero/KWNoseCone/1mNoseconeNormal.mbm').asNodeBuffer();
});
```

Other files in the archive seems to be readable OK.

JSZip version is the one provided on npm. This has been seen on Linux Mint (can't recall node version), and Fedora 20 (node 0.10.32).

I tried other `asSomething()` getters always running into the same error.

**Comment by seanfuture** [comment]
Encountered the same exact issue w/ ftp://ftp2.census.gov/geo/tiger/TIGER2015/ROADS/tl_2015_51109_roads.zip .. Looks like this bug is still present.

Thanks for tip @gregoire-astruc .. Downgrading to jszip@2.2.1 worked

**Comment by dduponchel** [comment]
I opened the issue https://github.com/nodeca/pako/issues/65 with the test case extracted from here.

**Comment by puzrin** [comment]
Released pako 0.2.8 with fix. If you reinstall jszip with dependencies, it should work now.

**Comment by dduponchel** [comment]
The release `v2.6.0` bundles this fix in the `dist/` files, fixing this bug everywhere.
